webpack.config.js

```diff
@@ -12,33 +12,51 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const path = require('path');
+
 module.exports = {
-  entry: './src/browser.js',
+  entry: './src/index.ts',
   output: {
-    library: 'scheduler',
-    filename: './scheduler.js',
+    library: 'CloudScheduler',
```

Comment:
`--main-service scheduler` will fix this

Reply:
thank you! fixing it